src/main/java/frc/robot/subsystems/DriveSubsystem.java

```diff
@@ -95,7 +125,13 @@ public void stopMotors() {
 		frontRightMotor.set(0);
 		backRightMotor.set(0);
 	}
+	public double getHeading() {
+		return -Math.IEEEremainder(gyro.getAngle(), 360);
```

Comment:
There's some minor quirks with how IEEEremainder handles negative values. Is there a reason you aren't using the regular remainder/modulo operation `gyro.getAngle() % 360`